lib/lbt/bundle/AutoSplitter.js

```diff
@@ -74,6 +74,25 @@ class AutoSplitter {
 					totalSize += "sap.ui.requireSync('');".length + toRequireJSName(module).length;
 				});
 				break;
+			case SectionType.DepCache:
+				moduleSizes["__depCacheSize"] = "sap.ui.loader.config({depCacheUI5:{}});".length;
```

Comment:
Maybe it makes sense to use a symbol here to prevent clashes with module names. I guess it's unlikely to happen but probably a good practice nevertheless.


```js
const depCacheSymbol = Symbol("depCache")
moduleSizes[depCacheSymbol] = ...
```

Reply:
Done